config/default.js

```diff
@@ -1,3 +1,3 @@
-export default {
+module.exports = {
```

Comment:
yes, use es6 it does not work constant port undefinded property and babel6 is module.exports[default]